### PR TITLE
This week in Redox 33.

### PR DESCRIPTION
**Pull request by goyox86**
Sorry by the delay! 

\o/

### CODE REVIEW

**Comment by jackpot51** [comment]
You may want to get rid of that DS_Store file

File: content/news/this-week-in-redox-33.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+
+Hello and welcome back to another issue of TWiR!
+
+First of all, apologies for disappearing for a while but I've been super busy at work these last few weeks.
+
+With that out of the way, let's start our tour!
+
+The fact that I was super busy these last couple of weeks did not mean that the rest of the team was idle too, in fact, they were super busy and because of that, I have a ton of progress to share!
+
+Lets's star with the main repo where [@sajattack](https://github.com/sajattack) and [@HarryU](https://github.com/HarryU) made some fixes to the `bootstrap.sh` script and [@dlrobertson](https://github.com/dlrobertson) made possible to keep debug info in a `.sym` file. Also, [@jackpot51](https://github.com/jackpot51) switched us to the `smolnetd` network stack!
```
Comment (jackpot51) [inline content/news/this-week-in-redox-33.md]: star -> start

Comment (jackpot51) [reply content/news/this-week-in-redox-33.md]: Also, "made possible to keep" -> "made it possible to keep"

File: content/news/this-week-in-redox-33.md [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+
+# PSA
+
+If you have any questions, ideas, or are curious about Redox, we recommend joining `#redox` on `irc.mozilla.org`, [our Discourse forum](https://discourse.redox-os.org/) or you can get an invite to our chat by sending an email request to `info@redox-os.org`.
+
+# What's new in Redox?
+
+## TL;DR
+
+Hello and welcome back to another issue of TWiR!
```
Comment (jackpot51) [inline content/news/this-week-in-redox-33.md]: TWiRx to differentiate it from This Week in Rust

File: content/news/this-week-in-redox-33.md [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+as well as documenting on how to use `gdb`.
+
+Keeping things low level (not really because Redox drivers are in userspace ;)), we have the **drivers** where [@jackpot51](https://github.com/jackpot51) made some `vesad` related updates in `ransid` and a fix of an error (`E0133`) on `pcid` was shipped by [@ghatdev](https://github.com/ghatdev).
+
+Also driver-related is **ransid**, the ANSI terminal driver, who got two new versions released (`0.4.4` and `0.4.5`) containing the ICH implementation and some fixes to DCH.
+
+The system call interface, AKA the **syscall** crate saw the birth of `fchown` and `fchmod` which should make porting easier.
+
+Before departing from the lower level layer we have **Redoxfs** with it's new major version, 3, with a new block size of `4096` bytes. This new version also includes an updated lock format, changes to set block size programmatically, a simplified disk cache and implementations of `fchmod` and `fchown`.
```
Comment (jackpot51) [inline content/news/this-week-in-redox-33.md]: Updated lock format should be removed, it is simply an update to Cargo.lock.

**Comment by goyox86** [comment]
Suggestions applied @jackpot51 and some redaction tweaks.

**Comment by jackpot51** [comment]
Thanks!

**Pull request merged**